spec/requests/cinema_halls_controller_spec.rb

```diff
@@ -13,6 +13,9 @@
     let(:cinema_hall) { create(:cinema_hall) }
 
     context 'when the user is logged in' do
+      let(:role) { create(:user_role, name: 'employee') }
+      let(:role) { create(:user_role, name: 'manager') }
```

Comment:
You often has it twice in your specs, probably should just left the one you want :)

Reply:
Refactored, thanks :)